examples/common/python/work_order_receipt/work_order_receipt_request.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import random
+
+import crypto.crypto as crypto
+import utility.signature as signature
+from error_code.error_status import ReceiptCreateStatus as ReceiptCreateStatus
+
+class WorkOrderReceiptRequest():
+    """
+    Class to create work order receipt APIs like
+    create, update, retrieve and lookup
+    """
+    def __init__(self):
+        self.sig_obj = signature.ClientSignature()
+        self.SIGNING_ALGORITHM = "SECP256K1"
+        self.HASHING_ALGORITHM = "SHA-256"
+        
+    def create_receipt(self, wo_request,
+        receipt_create_status, signing_key, nonce=None):
+        """
+        Create work order receipt corresponding to workorder id
+        Parameters:
+            - wo_request is jrpc work order request used to create the
+            work order request as defined in EEA spec 6.1.1.
+            - receipt_create_status is enum of type ReceiptCreateStatus
+            - signing_key is private key
+            - nonce is int type random number or monotonic counter
+        Returns jrpc request of type dictionary
+        """
+        final_hash_str = self.sig_obj.calculate_request_hash(wo_request)
+        wo_request_params = wo_request["params"]
+        worker_id = wo_request_params["workerId"]
+        requester_nonce = nonce
+        if nonce is None:
+            requester_nonce = str(random.randint(1, 10**10))
+        public_key =  signing_key.GetPublicKey().Serialize()
+        wo_receipt_reqest = {
```

Comment:
typo in variable name. change wo_receipt_reqest -> wo_receipt_request here and wherever used.

Reply:
Corrected.